src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -14,11 +16,21 @@ public class ListCommand extends Command {
 
     public static final String MESSAGE_SUCCESS = "Listed all contacts";
```

Comment:
Small detail, but it'd be better if this success message could be dynamically determined depending on the operation. `list --org` for instance could say `Listed all organisations` instead.

Reply:
True - it's been added.